### PR TITLE
v4.3.5 + autoupdate

### PR DESCRIPTION
**Pull request by orhtej2**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: manifest.toml [hunk 1]
```diff
@@ -54,9 +54,10 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://files.spip.net/spip/archives/spip-v4.3.3.zip"
-        sha256 = "e2a7e6094a398843b83d22f6ab4ed2433ceea8278ef183da0e324c31ef95ac60"
-        in_subdir = false
+        url = "https://git.spip.net/spip/spip/-/archive/v4.3.5/spip-v4.3.5.tar.bz2"
```
Comment (JamesRezo) [inline manifest.toml:57]:
```suggestion
        url = "https://files.spip.net/spip/archives/spip-v4.3.5.zip"
```

File: manifest.toml [hunk 2]
```diff
@@ -54,9 +54,10 @@ ram.runtime = "50M"
...
+        sha256 = "40e7f9d17dcb6e61a334757f745556853f864fe80c0476ee30526b4fa8c6c041"
```
Comment (JamesRezo) [inline manifest.toml:58]:
```suggestion
        sha1= "87002c7a8806b80f079ddbf6cf35c584986d999d"
```

**Review by JamesRezo — COMMENTED** [review]
We are providing urls and sha1 for releases at https://www.spip.net/spip_loader.api/3

File: manifest.toml (hunk 1 above)
Comment (rodinux) [reply manifest.toml:57]: Thanks for your help, in fact we can only use `sha256` for the sources.

Comment (orhtej2) [reply manifest.toml:57]: The change to get tag assets from git was intentional, this way autoupdater can pull new tags whenever a new version comes out.
This was to speed up updates in YNH, but OTOH I see a number of stale update attempts so I'll leave it to whoever decides to maintain the app.
If however we go with 'official' source file `in_subdir` and `autoupdate.*` are to be scrapped probably.

Comment (JamesRezo) [reply manifest.toml:57]: Well, the actual tag asset does not contain the whole source code of the app. 

@rodinux i can manage to provide a sha256 in the json i show you if needed

@orhtej2 we are in the process to provide a release arrtifact, but it'll take some weeks i guess

**Pull request closed without merging**